health/health.d/riakkv.conf

```diff
@@ -89,5 +92,6 @@ component: Riak KV
     every: 10s
      warn: $this > 10000
      crit: $this > 100000
-     info: number of processes running in the Erlang VM
+     info: Number of processes running in the Erlang VM. \
+           The number or processes is too high.
```

Comment:
We can’t conclusively determine that this is ‘too high’, though it’s very likely to be.

Once you get up to that point in process counts though, you start to see performance degradation due to scheduling overhead.